documentation/modules/metrics/proc-strimzi-metrics-reporter-kafka-deploy-options.adoc

```diff
@@ -0,0 +1,67 @@
+// This assembly is included in the following assemblies:
+//
+// metrics/assembly_metrics-kafka.adoc
+
+[id='proc-metrics-reporter-kafka-deploy-options-{context}']
+= Enabling Strimzi Metrics Reporter metrics through configuration
+
+[role="_abstract"]
+To enable and expose metrics in Strimzi for Prometheus, use metrics configuration properties.
+
+The following components require `metricsConfig` configuration to expose metrics in Prometheus format:
+
+* Kafka
+
+This configuration enables the {StrimziMetricsReporter} to expose metrics through an HTTP endpoint on port 9404.
+The port for the Strimzi Metrics Reporter HTTP endpoint is 9404.
+Prometheus scrapes this endpoint to collect Kafka metrics.
+
+To deploy Prometheus metrics configuration in Strimzi, you can use your own configuration or the xref:ref-metrics-reporter-config-{context}[example custom resource configuration file] provided with Strimzi:
+
+* `kafka-metrics.yaml`
+
+This file contain the necessary configuration to enable Prometheus metrics.
+It is a good starting point for trying Prometheus with Strimzi.
+
+This procedure shows how to deploy example Prometheus metrics configuration in the `Kafka` resource.
+The process is the same when deploying the example files for other resources.
```

Comment:
```suggestion

```

Reply:
we include a note/cross-ref on how to configure the Kafka Exporter as well in the JMX procedure. Do we want it here too?

Lines 28-31:  https://github.com/strimzi/strimzi-kafka-operator/pull/11307/files#diff-80eda6a9cce800533455d246c9b923ba06d53ea83645996177b63b5ef4f33e66R28-R31

Or maybe that needs to be separated from both now, so we link from each of these procedures. Note taken